tests/specs/guardian-redirects.spec.js

```diff
@@ -5,6 +5,7 @@ const { getRequest } = require('../utils/helpers');
 const { testScenarios } = require('../fixtures/scenarios');
 
 let GuardianSpecs;
+let ProductDetails;
```

Comment:
Two nit questions: 1. Why the CamelCase format here? Are these supposed to look like interface / class names? 2. Why are these in the global scope?

Reply:
I'm not sure why things were this way, but thanks for pointing them out. The fix extended beyond the scope of this PR so I have done it in #21.